swagger-java-client/src/test/java/io/dockstore/swagger/EnforceAnnotationsTest.java

```diff
@@ -18,8 +18,9 @@ public void checkAnnotations() {
         for (Annotation an : annotations) {
             final String toString = an.toString();
             if (toString.startsWith("@com.fasterxml.jackson.annotation.JsonSubTypes(") && toString
-                .contains("name=\"Service\", value=io.swagger.client.model.Service.class") && toString
-                .contains("name=\"BioWorkflow\", value=io.swagger.client.model.BioWorkflow.class")) {
+                    // https://stackoverflow.com/questions/15130309/how-to-use-regex-in-string-contains-method-in-java
```

Comment:
1. The reason you are changing this is because `Names` is getting inserted when it wasn't there before. Do you know why? I'm also unclear on the ramifications of this for the CLI. Probably none, but just want to make sure.
2. I find regex's hard to read, and it might be useful to show the string you are comparing against as a comment.

Reply:
I had to up the Jackson annotations version from 2.11.3 to 2.12.1 which added a new SubType.Type optional element 'names'. Compare:
https://fasterxml.github.io/jackson-annotations/javadoc/2.11/
and
https://fasterxml.github.io/jackson-annotations/javadoc/2.12/

Reply:
2. Added comment